ngraph/frontend/frontend_manager/src/frontend_manager.cpp

```diff
@@ -122,37 +132,15 @@ FrontEnd::FrontEnd() = default;
 
 FrontEnd::~FrontEnd() = default;
 
-InputModel::Ptr FrontEnd::load_from_file(const std::string& path) const
-{
-    FRONT_END_NOT_IMPLEMENTED(load_from_file);
-}
-
-InputModel::Ptr FrontEnd::load_from_files(const std::vector<std::string>& paths) const
-{
-    FRONT_END_NOT_IMPLEMENTED(load_from_files);
-}
-
-InputModel::Ptr FrontEnd::load_from_memory(const void* model) const
+bool FrontEnd::supported_by_variants(const std::vector<std::shared_ptr<Variant>>& variants) const
 {
-    FRONT_END_NOT_IMPLEMENTED(load_from_memory);
+    return false;
```

Comment:
don't we make a comment here about TODO? Now it seems incomplete method

Reply:
It is a complete method. It returns false if no implementation in frontend. Don't really think any more comments needed